### PR TITLE
[feat]: [CDS-86702]: Removing usages of FF POST_PROD_ROLLBACK

### PR DESCRIPTION
**Pull request by brijeshdhakar**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/56876)

### CODE REVIEW

**Review by archit-harness — DISMISSED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/56876)* status: 2 file(s) to review. 1 of 2 approvals obtained.

**Comment by sahilhindwani-harness** [comment]
*[`120-ng-manager/src/main/java/io/harness/ng/rollback/PostProdRollbackServiceImpl.java` line 41 at r1](https://reviewable.io/reviews/harness/harness-core/56876#-Nlvvn3T-sQbHdn8DjWu:-Nlvvn3T-sQbHdn8DjWv:bwu5ii2) ([raw file](https://github.com/harness/harness-core/blob/f2481698a9fd45c9757ca64c920ed128c373c808/120-ng-manager/src/main/java/io/harness/ng/rollback/PostProdRollbackServiceImpl.java#L41)):*
> ```Java
>   private static final Map<InstanceType, FeatureName> INSTANCE_TYPE_TO_FF_MAP =
>       Map.of(InstanceType.K8S_INSTANCE, POST_PROD_ROLLBACK, InstanceType.TAS_INSTANCE, POST_PROD_ROLLBACK,
>           InstanceType.ECS_INSTANCE, POST_PROD_ROLLBACK, InstanceType.ASG_INSTANCE, POST_PROD_ROLLBACK,
> ```

This check is required

**Comment by sahilhindwani-harness** [comment]
*[`120-ng-manager/src/main/java/io/harness/ng/rollback/PostProdRollbackServiceImpl.java` line 41 at r1](https://reviewable.io/reviews/harness/harness-core/56876#-Nlvvn3T-sQbHdn8DjWu:-NlvwJq84gdYb0_MlXQY:b-r3zwaj) ([raw file](https://github.com/harness/harness-core/blob/f2481698a9fd45c9757ca64c920ed128c373c808/120-ng-manager/src/main/java/io/harness/ng/rollback/PostProdRollbackServiceImpl.java#L41)):*
<details><summary><i>Previously, sahilhindwani-harness (Sahil Hindwani) wrote…</i></summary><blockquote>

This check is required
</blockquote></details>

Basically not all swimlanes gets onboarded to post prod rollback. This set holds what all swimlanes are supported for post prod rollback

**Review by brijeshdhakar — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/56876)* status: 2 file(s) to review. 1. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`120-ng-manager/src/main/java/io/harness/ng/rollback/PostProdRollbackServiceImpl.java` line 41 at r1](https://reviewable.io/reviews/harness/harness-core/56876#-Nlvvn3T-sQbHdn8DjWu:-Nlw2PQK8bGOqQelgHZA:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/f2481698a9fd45c9757ca64c920ed128c373c808/120-ng-manager/src/main/java/io/harness/ng/rollback/PostProdRollbackServiceImpl.java#L41)):*
<details><summary><i>Previously, sahilhindwani-harness (Sahil Hindwani) wrote…</i></summary><blockquote>

Basically not all swimlanes gets onboarded to post prod rollback. This set holds what all swimlanes are supported for post prod rollback
</blockquote></details>

Done.

**Review by archit-harness — APPROVED** [review]

**Review by sahilhindwani-harness — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/56876)* status: 1 file(s) to review.

**Comment by brijeshdhakar** [comment]
trigger review

**Comment by rohitk-h** [comment]
@brijeshdhakar can you please update the testing that was done as part of this change?

**Comment by brijeshdhakar** [comment]
@rohitk-h e2e testing has been done for k8s instance type in dev-space. Execution url for Post-prod-rollback: https://brijeshdhakar.pr2.harness.io/ng/account/nWv_iDS1QpeXKFff9_64Eg/module/cd/orgs/default/projects/Test/pipelines/deploy/deployments/6A6-t5-yRJCpNROXmygkMQ/pipeline
